### DISCUSSION TITLE
Fixed size message with a lot of data

### DISCUSSION THREAD

**Discussion by tunguskar** [intel/compile-time-init-build#630] [Q&A]
Hi, 

Unfortunately im still on the cpp17 branch. currently i want to define a message which is looking like this:

```cpp
  using FieldOne= msg::field<decltype("Start"_sc), 0, 31, 23, std::uint32_t, 0xFF>;
  using FieldLength = msg::field<decltype("Length"_sc), 0, 22, 8, std::uint32_t, 0x0102>;
  using FieldId = msg::field<decltype("ProductId"_sc), 0, 7, 0, std::uint32_t, 0x01>;
```

Now I want to add a space for a lot of data  bytes. I just wanted to know if there is an elegant way to add a field for example with 100 bytes of data?

Kind regards

**Comment by lukevalenty** [comment]
There is not currently a way to do that. The library was designed with fields less than 64-bits in size.
